data/json/items/gun/50.json

```diff
@@ -67,7 +67,9 @@
     },
     "material": [ "steel", "wood" ],
     "clip_size": 1,
+    "modes": [],
```

Comment:
I like 'manual' over 'semi', particularly since it's not a semi-automatic firearm.